packages/stark-ui/src/modules/app-logo/components/app-logo.component.ts

```diff
@@ -12,6 +12,7 @@ const componentName: string = "stark-app-logo";
 	selector: componentName,
 	templateUrl: "./app-logo.component.html",
 	encapsulation: ViewEncapsulation.None,
+	styleUrls: ['./app-logo.component.scss'],
```

Comment:
According to this, I see we scope the css into the component.
I'm not sure it's the way to go and the one we've talked about.
Could you please tell me more about this ? 😊

Reply:
Indeed, as we discussed, the styles should not be scoped, they should be global and they should be imported manually by the developer so he can decide whether to use our styles or not